### DISCUSSION TITLE
Please add more data fields from Xiaomi/Mi BLE devices to the json output via mqtt

### DISCUSSION THREAD

**Discussion by Jo-Bl** [arendst/Tasmota#20967] [Ideas]
### Discussed in https://github.com/arendst/Tasmota/discussions/20951

<div type='discussions-op-text'>

<sup>Originally posted by **Jo-Bl** March 14, 2024</sup>
Please add more data fields from Xiaomi/Mi BLE devices to the json output.

MCCGQ02HL devices (door sensors) sent a field called "packet". I want to have the value of it in mqtt. Please make an option so that it is possible to sent it. (reason: I want to have the possibility to check that I have received all packages with ascending numbers (0-255-0-255...))

The 'Mi Scale V1' device sent the fields  "weight removed" and "stabilized". I want to see this in json output via mqtt for my desition wether to get the weight value or not.

Jo 

</div>

**Comment by Jason2866** [comment]
Do NOT open an issue for an Feature Request.
